src/main/java/net/synedra/validatorfx/DefaultChecks.java

```diff
@@ -0,0 +1,285 @@
+    }
+
+    /**
+     * Create a check that fails validation when the string value of the passed property cannot be cast to a number.
+     *
+     * @param property whose value should be cast to a number
+     * @param severity of the validation
+     * @return self for methodChaining
+     */
+    public Check createIsNumberCheck(StringProperty property, Severity severity) {
+        return createIsNumberCheck(property, severity, null);
+    }
+
+    /**
+     * Create a check that fails validation when the number value of the passed property is not between two bounds.
+     *
+     * @param property  whose value should be between the bounds
+     * @param severity  of the validation
+     * @param decorated the node that should be decorated by this check
+     * @param minimum   the lower bound of the validation (inclusive)
+     * @param maximum   the upper bound of the validation (inclusive)
+     * @return self for methodChaining
+     */
+    public Check createIsNumberWithinBoundsCheck(StringProperty property, Severity severity, Node decorated, double minimum, double maximum) {
+        Function<String, Optional<Double>> isNumberWithinBoundsCheck = string -> Optional.ofNullable(asNumber(string)).filter(d -> d >= minimum && d <= maximum);
+        Function<String, String> messageCreator = string -> String.format("%s[%s] is not between %f and %f", property.getName(), Optional.ofNullable(asNumber(string)).map(String::valueOf).orElse("NAN"), minimum, maximum);
+        return createIsAssignableToCheck(property, severity, decorated, isNumberWithinBoundsCheck, messageCreator);
+    }
+
+    /**
+     * Create a check that fails validation when the number value of the passed property is not between two bounds.
+     *
+     * @param property whose value should be between the bounds
+     * @param severity of the validation
+     * @param minimum  the lower bound of the validation (inclusive)
+     * @param maximum  the upper bound of the validation (inclusive)
+     * @return self for methodChaining
+     */
+    public Check createIsNumberWithinBoundsCheck(StringProperty property, Severity severity, double minimum, double maximum) {
+        return createIsNumberWithinBoundsCheck(property, severity, null, minimum, maximum);
+    }
+
+    /**
+     * Create a check that fails validation when the string value of the passed property does not match a regex.
+     *
+     * @param property  whose value should match the regex
+     * @param severity  of the validation
+     * @param decorated the node that should be decorated by this check
+     * @param regex     that the property-value should match
+     * @return self for methodChaining
+     */
+    public Check matchesRegexCheck(StringProperty property, Severity severity, Node decorated, String regex) {
+        Function<String, String> messageCreator = string -> String.format("%s['%s'] does not match the regex %s", property.getName(), string, regex);
```

Comment:
This is a problem for i18n. We must not provide any natural language strings from within ValidatorFX.